### ISSUE TITLE
import/no-unresolved

### ISSUE THREAD

**Issue by xereda** [benmosher/eslint-plugin-import#945]
I am using the vscode editor and am having problems with eslint validation of false-positive type. My operating system mac os (high sierra).

More details:

Error:
```
[eslint] Casing of [relative path] does not match the underlying filesystem. (import / no-unresolved)
module "[module path]"
```

![captura de tela 2017-10-13 as 12 34 17](https://user-images.githubusercontent.com/14337850/31554107-f03d6336-b012-11e7-890c-1c9d55eb28f2.png)

![captura de tela 2017-10-13 as 12 34 22](https://user-images.githubusercontent.com/14337850/31554110-f36307d2-b012-11e7-8aaf-b50fb07b2a83.png)

![captura de tela 2017-10-13 as 12 34 27](https://user-images.githubusercontent.com/14337850/31554116-f65ffae4-b012-11e7-8af0-c5000bb53476.png)

![pajzvoej82](https://user-images.githubusercontent.com/14337850/31554582-5e76a35c-b014-11e7-9b46-fabdc91ea6b4.gif)

    "eslint": "^4.8.0",
    "eslint-config-airbnb": "^15.0.1",
    "eslint-import-resolver-webpack": "^0.8.3",
    "eslint-plugin-import": "^2.6.0",
    "eslint-plugin-jest": "^21.2.0",
    "eslint-plugin-jsx-a11y": "^6.0.2",

**Comment by wachunei** [comment]
I guess this is related with #720

**Comment by M1chaelTran** [comment]
+1

**Comment by Sage911** [comment]
@xereda Are you opening VS Code from the command line? I've seen this error come up in Atom when I opened a project using by cd'ing using the wrong casing. ea: desktop/projectname vs Desktop/projectname

**Comment by xereda** [comment]
@Sage911 I open the project from within the vscode.

**Comment by judgejab** [comment]
I'm having the exact same issue with Atom.

**Comment by pcguru** [comment]
Getting the same problem. `./node_modules/.bin/eslint .` works fine from command line, but does not work in terminal in Visual Studio Code. Does not work with the ESLint plugin either - all imports in code files are marked with this error.

**Comment by ljharb** [comment]
@pcguru That seems like an issue with VSCode. The only thing that matters is that it works on the command line; anything else is an editor issue.

I'm going to close this; there doesn't seem to be any issue here that's specific to this plugin.

**Comment by EdwardSalter** [comment]
I am seeing this when via command line when running it through a setup similar to how create-react-app works.
We have a project that houses our webpack build alongside our application boilerplate. I was seeing the error in this app as well until I upgraded all eslint dependencies and it has now disappeared.

After creating a new app using the boilerplate (which is CRA compatible), the new application suffers this problem. This occurs when running eslint via the build script (which uses eslint-loader inside of webpack) and also when running via command line (`npx eslint src`). The plugin inside of Atom is not affected by this issue for some reason.

**Comment by yoyo837** [comment]
It report error ✖️: 
`Casing of ./FormCreate does not match the underlying filesystem  import/no-unresolved`
```js
/* eslint-disable import/prefer-default-export */
import FormCreate from './FormCreate';

export { FormCreate };

```

It does not report error ☑️: 
```diff
/* eslint-disable import/prefer-default-export */
+
import FormCreate from './FormCreate';

export { FormCreate };

```

**Comment by ljharb** [comment]
@yoyo837 please file a new issue.
